llvm/lib/Target/AMDGPU/AMDGPURewriteAGPRCopyMFMA.cpp

```diff
@@ -259,13 +259,15 @@ bool AMDGPURewriteAGPRCopyMFMAImpl::run(MachineFunction &MF) const {
       LLVM_DEBUG({
         dbgs() << "Attempting to replace VGPR MFMA with AGPR version:"
                << " Dst=[" << printReg(VReg) << " => "
-               << printReg(PhysReg, &TRI) << ']';
+               << printReg(PhysReg, &TRI);
```

Comment:
Missing brace at end of `Dst=[` in the case that we have a Src2Reg

I'm seeing:
`Attempting to replace VGPR MFMA with AGPR version: Dst=[%7 => $agpr0_agpr1_agpr2_agpr3, Src2=[%2 => $vgpr2_vgpr3_vgpr4_vgpr5]`